war/src/main/resources/templates/scores/details.html

```diff
@@ -90,7 +93,23 @@ <h3>Probes results</h3>
                     <ion-icon data-th-if="${probeResult.status.name} == 'SUCCESS'" class="success" name="success"></ion-icon>
                     <ion-icon data-th-unless="${probeResult.status.name} == 'SUCCESS'" class="failure" name="failure"></ion-icon>
                 </td>
-                <td data-th-text="${probeResult.message}"></td>
+                 <td>
+                    <span
+                        th:if="${probeResult.id == 'security'}">
```

Comment:
So far, the tag used were using the HTML 5 format: `<data-th-*/>`. Please use that.

Reply:
Please, don't repeat same mistake.